### DISCUSSION TITLE
addPeak2GeneLinks using normalized/corrected data, account for GC content?

### DISCUSSION THREAD

**Discussion by agreco92** [GreenleafLab/ArchR#1212] [Questions / Documentation]
Hello,

I am exploring the possibility of creating a Gene Regulatory Network starting from the addPeak2GeneLinks in a multiomic RNA-ATAC dataset. However, my RNA was processed in such a way to remove some technical biases using Seurat (thus the relevant data is in the @scale.data slot). Unfortunately, the addGeneExpressionMatrix function only accepts integer-valued count matrices, that in my case is not corrected for any technical bias. Is there any way to work around this limitation and use the scaled data instead?

Secondarily, I love the idea of adding an empirical p-value to Peak2Gene links. However, in [this paper](https://www.sciencedirect.com/science/article/pii/S0092867420312538), authors recommend to correct for GC content in addition to sampling depth. Given that ArchR is also generating background peaks accounting for GC content (addBgdPeaks function), is there a way to use these ones as background peaks for the computation of correlations?

Thank you for your help (and for the fantastic package!)

**Comment by rcorces** [comment]
> Is there any way to work around this limitation and use the scaled data instead?

Currently no

> Secondarily, I love the idea of adding an empirical p-value to Peak2Gene links. However, in this paper, authors recommend to correct for GC content in addition to sampling depth. Given that ArchR is also generating background peaks accounting for GC content (addBgdPeaks function), is there a way to use these ones as background peaks for the computation of correlations?

I'm not sure I understand your idea but feel free to make a more detailed feature request.
